README.md

```diff
@@ -160,7 +160,7 @@ The remaining options are passed directly along to `toil`:
 
 ## Autoscale
 
-To use the autoscale functionality
+To use the autoscale functionality, change the resources to `toil = 1` in `luigi.cfg`
```

Comment:
@joelarmstrong Are you OK with it being like this, or do you want to go the opposite way?

Reply:
I think this way is better (keeping it at 5 for most users, and putting the responsibility on autoscaler users to change it). In an ideal world, the toil autoscaler would eventually get fixed to avoid this anyway.